platform/golden/bullseye.pkr.hcl

```diff
@@ -1,59 +1,107 @@
-variables {
-  headless          = true
-  iso_checksum      = "224cd98011b9184e49f858a46096c6ff4894adff8945ce89b194541afdfd93b73b4666b0705234bd4dff42c0a914fdb6037dd0982efb5813e8a553d8e92e6f51"
-  iso_checksum_type = "sha512"
-  iso_url           = "https://cdimage.debian.org/debian-cd/11.6.0/amd64/iso-cd/debian-11.6.0-amd64-netinst.iso"
+variable "config_file" {
+  type    = string
+  default = "debian-11-preseed.cfg"
+}
+
+variable "config_dir" {
+  type    = string
+  default = "cfg"
+}
+
+variable "cpus" {
+  type    = string
+  default = "4"
+}
+
+variable "disk_size" {
+  type    = string
+  default = "4096"
```

Comment:
what is the thinking here? I am good with a smaller boot volume, although [OCI requires atleast 50GB](https://docs.oracle.com/en-us/iaas/Content/Block/Concepts/bootvolumes.htm#:~:text=When%20you%20launch%20an%20instance,size%20up%20to%2032%20TB.).

We can always tune the size later, leaving a comment here for that purpose.

Reply:
Was just to ensure that I had enough disk space on a test instance. I wasn't sure if CoW was being implemented, so wanted to make sure that disk size wasn't causing issues.  Made it 64Gb